photon-lib/src/main/scala/com/linkedin/photon/ml/util/VectorUtils.scala

```diff
@@ -234,4 +240,16 @@ object VectorUtils {
     case _ => vector.activeKeysIterator.toSet
   }
 
+  /**
+    * We need to migrate away from mllib.linalg.Vector but there are still places where it cannot be replaced yet
+    * This is basically a copy of mllib.linalg.Vector.fromML, so that we do not need to import mllib.linalg.Vector
+    * at other places
+    * @param v the ml.linalg.Vector
+    * @return mllib.linalg.Vector (either dense or sparse)
+    */
+  def mlVectorToMllibVector(v: SparkMLVector): SparkVector = v match {
+    case dv: SparkMLDenseVector => SparkDenseVector.fromML(dv)
+    case sv: SparkMLSparseVector => SparkSparseVector.fromML(sv)
+  }
```

Comment:
I think for this use-case, we should keep the import to `spark.mllib.linalg.Vectors`, since for things such as the `Statistics` object, Spark does not currently provided an alternative. Let's drop this function and keep the call to `spark.mllib.linalg.Vectors`.